openc3-cosmos-init/plugins/packages/openc3-tool-base/src/router.js

```diff
@@ -33,22 +32,22 @@ const router = createRouter({
       name: 'Login',
       component: () =>
         import(
-          '../../packages/openc3-tool-common/src/tools/base/components/Login'
+          '@openc3/tool-common/src/tools/base/components/Login'
         ),
     },
     {
       // Empty component for all other routes to avoid VueRouter warnings, since all other routes are handled by single-spa
       path: '/:pathMatch(.*)*',
       name: '',
       component: () =>
-        import('../../packages/openc3-tool-common/src/components/Empty'),
+        import('@openc3/tool-common/src/components/Empty'),
     },
   ],
 })
 
 router.beforeEach(({ path }) => {
   if (['/', '/tools', '/tools/'].includes(path)) {
-    navigateToUrl(DEFAULT_TOOL_PATH)
+    singleSpaNavigate(DEFAULT_TOOL_PATH)
```

Comment:
AppNav.vue does a bunch of `navigateToUrl` ... does it also need to change to `singleSpaNavigate`?

Reply:
As far as I understand, they are interchangeable. It's just `navigateToUrl` has to be imported while `singleSpaNavigate` is just added as a global via the `window` object. I had changed them from singleSpaNavigate _to_ navigateToUrl during the initial vue 3 migration stuff (while I was trying to get single-spa actually working), but that change should just be vestigial by now.

https://single-spa.js.org/docs/api/#navigatetourl